pandas/tests/extension/json/array.py

```diff
@@ -27,7 +28,7 @@
 class JSONDtype(ExtensionDtype):
     type = abc.Mapping
     name = "json"
-    na_value = UserDict()
+    na_value = UserDict()  # type: Mapping[type, type]
```

Comment:
I don't think `type` should be used for subscripts; I think just `Mapping[str, Any]` is fine

Reply:
will need to change to py3.6 syntax.